#This maneuver follows a reference given by an external entity.
imc_ros_interface/Header header
std_msgs/UInt16 control_src
std_msgs/UInt8 control_ent
std_msgs/Float32 timeout
std_msgs/Float32 loiter_radius
std_msgs/Float32 altitude_interval
